### ISSUE TITLE
FCC Challenge Function

### ISSUE THREAD

**Issue by jodylecompte** [campDevs/DiscordBot#56]
With the release of the new FCC curriculum and subdomain, it is fairly safe to assume our scraper based FCC challenge function is now broken.

**Comment by glynnw** [comment]
Yeah well it isn't a very powerful command. You had to provide the actual link to the challenge and such. We could probably do better as well.

**Comment by pahosler** [comment]
is there anything we could pull from their repo to make it easier? There are so many new challenges now, and additional sections with even more challenges. The Coding Interview Prep challenges alone have hundreds and hundreds of coding challenges.
